.github/workflows/validate-prs.yml

```diff
@@ -35,12 +36,17 @@ jobs:
             const title = "${{env.TITLE}}";
             const body = context.payload.pull_request.body;
             const labels = "${{env.LABELS}}";
+            const author = "${{env.AUTHOR}}";
+
+            const excludeAuthors = ["dependabot", "LTA-Thinking"];
             
             if(title.toLowerCase().includes('personal') || (title.length > 65)) {
               errors += '- Title of the PR to be succinct and less than 65 characters. \n';
             }
 
-            if(body.toLowerCase().includes('ab#') == false && body.toLowerCase().includes('/_workitems') == false) {
+            if(body.toLowerCase().includes('ab#') == false &&
+               body.toLowerCase().includes('/_workitems') == false &&
+               !excludeAuthors.includes(author)) {
```

Comment:
I was thinking we can probably just ignore the label "Dependencies"?

Reply:
The other issue I'm trying to fix here is external contributors. We have an external contribution just waiting right now because we don't have a work item for it and I don't think it makes sense to create one.

Reply:
Sergey recommended having a list of excluded authors instead of a tag system to avoid sidestepping the requirement.